wpg/pg.go

```diff
@@ -41,6 +41,10 @@ type Column struct {
 	Type string `db:"data_type"json:"type"`
 }
 
+func escaped(s string) string {
```

Comment:
i guess some optimization _could_ be checking if the column name == a [postgres reserved keyword](https://www.postgresql.org/docs/current/sql-keywords-appendix.html) but there are so many of those. plus would need to be maintained as postgres adds keywords. `"` should ~always work.

Reply:
Hrm. It's kind of annoying to have everything quoted. I wonder if we can build the set of reserved words on startup using this query:

```sql
select word from pg_get_keywords() where catdesc = 'reserved';
```

Reply:
i thought about that, but doesn't that impose limitations such as requiring a pg connection to run shovel with the `-schema` flag

Reply:
imo i think if we check if it's in the reserved word list, best to just bake it into the code and update as needed

Reply:
I'm fine with that too. Perhaps we can use `go generate` to create the list. I don't think this will change too much.